### DISCUSSION TITLE
Metal support for macOS

### DISCUSSION THREAD

**Discussion by ltecheroffical** [godotengine/godot-proposals#6841] [Platforms]
This would be a setting in the macOS export template to allow you to use metal to render the game, it would speed up the output and be better for intense apps.

**Comment by AThousandShips** [comment]
See https://github.com/godotengine/godot/pull/72831

**Comment by stuartcarnie** [comment]
See https://github.com/godotengine/godot/pull/88199

**Reply by amirtheswarann** [reply]
hi @stuartcarnie  do you have any plans on metal support for x86_64 Mac in the future ?

**Reply by Calinou** [reply]
This was explored in https://github.com/godotengine/godot/pull/96158, but it's not currently planned as the Metal renderer relies on features only present on Apple Silicon hardware. Adding support for x86_64 would mean a lot of fallback paths would have to be implemented, which would also reduce the performance benefit of using Metal instead of MoltenVK.

In any case, MoltenVK and the Compatibility rendering method will keep supporting x86_64 macOS for the foreseeable future.
